### ISSUE TITLE
LoadStream appearing to not load some mpvinewood streams

### ISSUE THREAD

**Issue by Qwerty1Verified** [citizenfx/fivem#1590]
Attempting to load and play streams from the mpvinewood update and the only thing that seems to work is `LoadStream('casino_walla', 'dlc_vw_casino_interior_sounds')`, I searched through the mpvinewood update REL files to see if I could find anything relating to "casino_walla" individually and I could only find things like `dlc_vinewood_casino_walla_stemmed_interior_walla_w_slots`.

I searched further into the sounds REL file and found a stream with the name `dlc_vinewood_casino_walla_stemmed_tone_stream2`, but when attempting to play it, the `LoadStream` native didn't resolve true at any point, I did even try to cut parts of the name off of the stream name and that didn't work either.

I also noticed issues when attempting to use `RequestScriptAudioBank`, I could only get one audio bank to not hang when waiting for it to be requested and that was `DLC_VINEWOOD/CASINO_GENERAL`, when trying to request `DLC_VINEWOOD/CASINO_INTERIOR_STEMS` it would hang waiting.

**Comment by Qwerty1Verified** [comment]
I'm slightly unsure on whether the data I'm pulling out of the sounds REL files is the correct data because I tried to play music from the base game radio stations and couldn't do that either. The data I used was:
```
  <Item type="StreamingSound">
   <Name>hash_E11F9385</Name>
   <Header>
    <Flags value="0x0180C001" />
    <Flags2 value="0xAA90AAAA" />
    <DopplerFactor value="0" />
    <Category>hash_45EB536F</Category>
    <Unk20 value="0" />
    <Unk21 value="1" />
   </Header>
   <Duration value="205898" />
   <ChildSounds>
    <Item>hash_6BD3AE84</Item>
    <Item>hash_97EAF501</Item>
   </ChildSounds>
  </Item>
  ```
I took the name hoping I could use the hash name as the stream name considering it has a type of "StreamingSound", and tried to use both `0` and `radio_02_pop` as the soundset and none worked. I also tried following the child sounds hashes to get the actual name of the song and use that in place as the stream and that didn't work either. I'm unsure whether it's me failing to scrape the correct data from the game files (plausible due to the low documentation amount) or if it's down to a bug
